pages/self-service.html

```diff
@@ -74,9 +74,8 @@ <h2 id="riderConfirmedHeader" class="self-service-heading">Rider Confirmed Match
 
                     <div class="form-group">
                         <button class="button button--danger" id="btnCancelRideRequest">Cancel Ride Request</button>
-                        <button class="button button--danger" id="btnCancelRiderMatch">Cancel Rider Match Request</button>
```

Comment:
This merging of the two cancel functions and buttons looks like a very promising refactor. I've probably not got time to check it before the Nov 7 election, and the changes to element ids may stop my automated tests from working. With the limited time and support team numbers, I'd like to keep those tests going if possible. Can we put this change to one side until after Nov 7, please?

Reply:
Sure, let's update the tests when we merge after Nov 7.